nav2_smac_planner/src/smac_planner_2d.cpp

```diff
@@ -292,6 +295,10 @@ nav_msgs::msg::Path SmacPlanner2D::createPlan(
     _smoother->smooth(plan, costmap, time_remaining);
   }
 
+  if (!_ignore_goal_orientation && plan.poses.size() > 0) {
+    plan.poses[plan.poses.size() - 1].pose.orientation = goal.pose.orientation;
```

Comment:
This can go in the Hybrid as well!

Reply:
and `back()`

Reply:
I did not did into the hybrid version yet but it according to my experiment it is not needed for it to respect the goal orientation: https://github.com/ros-planning/navigation2/issues/2482#issuecomment-888133212

Reply:
Got it, makes sense